connect/runtime/src/test/java/org/apache/kafka/connect/util/clusters/EmbeddedConnectClusterAssertions.java

```diff
@@ -16,603 +16,14 @@
-        try {
-            int numRunning = connect.kafka().runningBrokers().size();
-            return Optional.of(comp.apply(numRunning, numBrokers));
-        } catch (Exception e) {
-            log.error("Could not check running brokers.", e);
-            return Optional.empty();
-        }
-    }
-
-    /**
-     * Assert that the topics with the specified names do not exist.
-     *
-     * @param topicNames the names of the topics that are expected to not exist
-     */
-    public void assertTopicsDoNotExist(String... topicNames) throws InterruptedException {
-        Set<String> topicNameSet = new HashSet<>(Arrays.asList(topicNames));
-        AtomicReference<Set<String>> existingTopics = new AtomicReference<>(topicNameSet);
-        waitForCondition(
-            () -> checkTopicsExist(topicNameSet, (actual, expected) -> {
-                existingTopics.set(actual);
-                return actual.isEmpty();
-            }).orElse(false),
-            CONNECTOR_SETUP_DURATION_MS,
-            "Unexpectedly found topics " + existingTopics.get());
-    }
-
-    /**
-     * Assert that the topics with the specified names do exist.
-     *
-     * @param topicNames the names of the topics that are expected to exist
-     */
-    public void assertTopicsExist(String... topicNames) throws InterruptedException {
-        Set<String> topicNameSet = new HashSet<>(Arrays.asList(topicNames));
-        AtomicReference<Set<String>> missingTopics = new AtomicReference<>(topicNameSet);
-        waitForCondition(
-            () -> checkTopicsExist(topicNameSet, (actual, expected) -> {
-                Set<String> missing = new HashSet<>(expected);
-                missing.removeAll(actual);
-                missingTopics.set(missing);
-                return missing.isEmpty();
-            }).orElse(false),
-            CONNECTOR_SETUP_DURATION_MS,
-            "Didn't find the topics " + missingTopics.get());
-    }
+@Deprecated
+public class EmbeddedConnectClusterAssertions extends ConnectAssertions {
 
-    protected Optional<Boolean> checkTopicsExist(Set<String> topicNames, BiFunction<Set<String>, Set<String>, Boolean> comp) {
-        try {
-            Map<String, Optional<TopicDescription>> topics = connect.kafka().describeTopics(topicNames);
-            Set<String> actualExistingTopics = topics.entrySet()
-                                                     .stream()
-                                                     .filter(e -> e.getValue().isPresent())
-                                                     .map(Map.Entry::getKey)
-                                                     .collect(Collectors.toSet());
-            return Optional.of(comp.apply(actualExistingTopics, topicNames));
-        } catch (Exception e) {
-            log.error("Failed to describe the topic(s): {}.", topicNames, e);
-            return Optional.empty();
-        }
+    public EmbeddedConnectClusterAssertions(EmbeddedConnect connect) {
```

Comment:
nit: the constructor was package-private before
```suggestion
   EmbeddedConnectClusterAssertions(EmbeddedConnect connect) {
```